src/lib/core/ripple/ripple.spec.ts

```diff
@@ -111,15 +113,29 @@ describe('MdRipple', () => {
     }));
 
     it('creates ripples when manually triggered', () => {
-      let rippleComponent = fixture.debugElement.componentInstance.ripple as MdRipple;
-
       expect(rippleTarget.querySelectorAll('.mat-ripple-element').length).toBe(0);
 
-      rippleComponent.launch(0, 0);
+      rippleDirective.launch(0, 0);
 
       expect(rippleTarget.querySelectorAll('.mat-ripple-element').length).toBe(1);
     });
 
+    it('creates manual ripples with the default ripple config', () => {
+      expect(rippleTarget.querySelectorAll('.mat-ripple-element').length).toBe(0);
```

Comment:
Yeah it might be fractionally faster, but I think when checking for ripple elements we want to explicitly check for an amount. Therefore the falsy check should be consistent here.